### ISSUE TITLE
icon files do not change in systray

### ISSUE THREAD

**Issue by farseerfc** [qTox/qTox#3131]
##### Brief Description

OS: Arch Linux (rolling)
qTox version:  v1.3.0-170-g0a2f541
Commit hash:  0a2f54194569e626ef8501c610647219887505d4
toxcore:  0.0.0 (3747-1 in Arch Linux)
Qt:  5.6.0

This is the same issue as https://bugs.archlinux.org/task/47910
Keeping the icons as qtox.png / qtox.svg breaks the change of tray icon based on status.

Reproducible: Always
##### Steps to reproduce
1. Start qtox 
2. login
##### Observed Behavior

icon in systray is still a black lock, the default icon [blacklock](https://raw.githubusercontent.com/tux3/qTox/master/img/icons/24x24/qtox.png)
##### Expected Behavior

icon in systray should change to online icon [online](https://raw.githubusercontent.com/tux3/qTox/master/img/taskbar/dark/taskbar_online.svg)
##### Additional Info

See https://bugs.archlinux.org/task/47910

A workaround will be to rename icons from qtox.\* to qTox.*

**Comment by zetok** [comment]
> icon in systray is still a black lock, the default icon blacklock

Related: https://github.com/tux3/qTox/issues/1820#issuecomment-133818693

After some update of Qt 5.5.1 it's fixed for me.

Could you perhaps test whether with different Qt version (e.g. 5.4) there is still problem with lower-case `qtox.*` icons?

**Comment by farseerfc** [comment]
I will try to test on an older version of Qt but I need some time to setup the environment (because my main working DE is KDE5 and I can't risk to break it).

**Comment by vith** [comment]
I get the bug on release builds of Qt from Ubuntu or Arch Linux packages but not debug builds from Qt git tags v5.5.0, v5.5.1, or v5.6.0. Haven't been able to properly debug the release builds to see what causes the difference but I created this workaround for now: https://github.com/tux3/qTox/pull/3151

**Comment by zetok** [comment]
Since #3151 was merged, this should be fixed.
@farseerfc care to test with newest qTox?

**Comment by farseerfc** [comment]
Wow tested ed9ccbe4716930ff27044201480589033c5d3b6f and it works!
Thank you @vith and @zetok .
I will close this issue.

PS:
@vith I added you as the co-maintainer of qtox-git on AUR. Feel free to update it if necessary. And let me know if this troubles you.

PPS:
I have a [nightly build robot](https://github.com/archlinuxcn/repo/commit/d72b7da07c3fd082c587c56d26f06b42e8f1215e) to build the master branch (and push the updated PKGBUILD to AUR if the build is success) everyday, and I tested it manually whenever I saw a new build.
